### PR TITLE
Enforce more eslint rules

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

We had some PRs recently where we were wondering why we didn't enforce certain code styles a bit more. I couldn't find the PR anymore but I went through the common eslint style definitions and enforced some rules that were used across the codebase already.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,27 +1,41 @@
...
-    "import/namespace": 2,
-    "import/default": 2,
-    "import/export": 2
+    'import/no-unresolved': [2, {commonjs: true, amd: true}],
+    'import/named': 2,
+    'import/namespace': 2,
+    'import/default': 2,
+    'import/export': 2,
+
+    'no-multiple-empty-lines': [2, {'max': 2, 'maxEOF': 1}],
```
Comment (mgrybyk) [inline .eslintrc.js]: @christian-bromann however this allows two lines
```
const waitForExist = require('../../src/commands/element/waitForExist')


describe('refetchElement', () => {
```
will be ignored

File: .eslintrc.js [hunk 2]
```diff
@@ -1,27 +1,41 @@
 module.exports = {
```
Comment (mgrybyk) [inline .eslintrc.js]: we have lines like `{ allowSingleLine: true }` and `{commonjs: true, amd: true}`. Can we use same style space-in-brackets style?

Comment (christian-bromann) [reply .eslintrc.js]: I will set `space-in-brackets` to always because it has less errors (550 vs 2400)

File: .eslintrc.js [hunk 3]
```diff
@@ -1,27 +1,42 @@
...
+    semi: ['error', 'never'],
+    quotes: ['error', 'single'],
     indent: [2, 4],
 
-    "import/no-unresolved": [2, {commonjs: true, amd: true}],
-    "import/named": 2,
-    "import/namespace": 2,
-    "import/default": 2,
-    "import/export": 2
+    'import/no-unresolved': [2, {commonjs: true, amd: true}],
```
Comment (mgrybyk) [inline .eslintrc.js]: Minor styling issue, can you reformat this file?

File: .eslintrc.js [hunk 4]
```diff
@@ -1,27 +1,42 @@
...
+    'import/named': 2,
+    'import/namespace': 2,
+    'import/default': 2,
+    'import/export': 2,
+
+    'no-multiple-empty-lines': [2, {'max': 1, 'maxEOF': 1}],
```
Comment (mgrybyk) [inline .eslintrc.js]: Maybe I missed your answer,  what do you think on changing lines to 1?

File: .eslintrc.js (hunk 3 above)
Comment (christian-bromann) [reply .eslintrc.js]: reformat into what?

File: .eslintrc.js (hunk 4 above)
Comment (christian-bromann) [reply .eslintrc.js]: Than we would only get warnings if someone breaks that rule.

File: .eslintrc.js (hunk 3 above)
Comment (mgrybyk) [reply .eslintrc.js]: The file itself. There are issues with spaces in curly bracers

Comment (christian-bromann) [reply .eslintrc.js]: mhm for some reason this wasn't caught

**Pull request merged**